composeApp/src/commonMain/kotlin/sp/bvantur/tasky/login/ui/LoginScreen.kt

```diff
@@ -136,4 +147,18 @@ fun LoginScreen(viewState: LoginViewState, onNavigateToRegister: () -> Unit, onU
             }
         }
     }
+
+    if (viewState.showErrorDialog) {
+        TaskyErrorDialog(
+            title = stringResource(Res.string.error_with_login),
+            message = stringResource(Res.string.error_general_message),
+            onDismissAction = {
+                onUserAction(LoginUserAction.DismissErrorDialog)
+            },
+            onConfirmAction = {
+                onUserAction(LoginUserAction.DismissErrorDialog)
+            }
+
+        )
```

Comment:
Something small that I’ve found useful is to instead model error states as a nullable message so that it is possible to easily show different error dialogs with different messages. It functions the same as using a boolean since you only show it when you have a message, else not when it is `null`, but gives you the flexibility to easily add different error messages.